CK3toEU4/Source/Mappers/CultureDefinitionsMapper/CultureDefinitionsMapper.cpp

```diff
@@ -0,0 +1,170 @@
+#include "CultureDefinitionsMapper.h"
+#include "../../CK3World/Cultures/Culture.h"
+#include "../../Configuration/Configuration.h"
+#include "../CultureDefinitionsMapper/CultureDefiniton.h"
+#include "../CultureMapper/CultureMapper.h"
+#include "CommonRegexes.h"
+#include "CultureGroupDefinition.h"
+#include "Log.h"
+#include "OSCompatibilityLayer.h"
+#include "ParserHelpers.h"
+#include <ranges>
+
+void mappers::CultureDefinitionsMapper::initForEU4(const Configuration& theConfiguration)
+{
+	Log(LogLevel::Info) << "Parsing Cultures and Culture Groups";
+	registerKeys();
+
+	for (const auto& cultureFile: commonItems::GetAllFilesInFolder(theConfiguration.getEU4Path() + "/common/cultures/"))
+		parseFile(theConfiguration.getEU4Path() + "/common/cultures/" + cultureFile);
+
+	for (const auto& cultureFile: commonItems::GetAllFilesInFolder("blankMod/output/common/cultures/"))
+		parseFile("blankMod/output/common/cultures/" + cultureFile);
+
+	clearRegisteredKeywords();
+}
+
+mappers::CultureDefinitionsMapper::CultureDefinitionsMapper(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+void mappers::CultureDefinitionsMapper::registerKeys()
+{
+	registerRegex(R"([\w_]+)", [this](std::string cultureGroupName, std::istream& theStream) {
```

Comment:
Wouldn't a string regex from commonRegexes work here?

Reply:
Probably, I'm unsure, though this is chars and digits plus underscore, which is what cultures use. Can go either way I guess.